### PR TITLE
Added upgrade tool docs

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _upgrade-to/upgrade-to.md [hunk 1]
```diff
@@ -179,3 +179,97 @@ If you are upgrading an Open Distro for Elasticsearch cluster, we recommend firs
...
+   {
+     "persistent": {
+       "cluster.routing.allocation.enable": "primaries"
+     }
+   }
+   ```
+
+1. On any one of the nodes, download and extract the OpenSearch tarball to a new directory.
+
+1. Make sure the following environment variables are set:
```
Comment (aetter) [inline _upgrade-to/upgrade-to.md:221]: I think it's worth including an example export command here for how to set one of these.

File: _upgrade-to/upgrade-to.md [hunk 2]
```diff
@@ -179,3 +179,97 @@ If you are upgrading an Open Distro for Elasticsearch cluster, we recommend firs
...
+    - `OPENSEARCH_HOME` - Path to the OpenSearch installation home.
+    - `OPENSEARCH_PATH_CONF` - Path to the OpenSearch config directory.
+
+1. The `opensearch-upgrade` tool is in the `bin` directory of the distribution. Run the following command from the distribution home:
+
+    Make sure you run this tool as the same user running the current Elasticsearch service.
+    {: .note }
+
+   ```json
+     ./bin/opensearch-upgrade
```
Comment (aetter) [inline _upgrade-to/upgrade-to.md:234]: Indentation issue, can pull the command back two spaces.

File: _upgrade-to/upgrade-to.md [hunk 3]
```diff
@@ -179,3 +179,97 @@ If you are upgrading an Open Distro for Elasticsearch cluster, we recommend firs
...
+1. Make sure the following environment variables are set:
+
+    - `ES_HOME` - Path to the existing Elasticsearch installation home.
+    - `ES_PATH_CONF` - Path to the existing Elasticsearch config directory.
+    - `OPENSEARCH_HOME` - Path to the OpenSearch installation home.
+    - `OPENSEARCH_PATH_CONF` - Path to the OpenSearch config directory.
+
+1. The `opensearch-upgrade` tool is in the `bin` directory of the distribution. Run the following command from the distribution home:
+
+    Make sure you run this tool as the same user running the current Elasticsearch service.
```
Comment (aetter) [inline _upgrade-to/upgrade-to.md:230]: One space back so that everything under the step lines up.

**Review by aetter — APPROVED** [review]

File: _upgrade-to/upgrade-to.md (hunk 1 above)
Comment (ashwinkumar12345) [reply _upgrade-to/upgrade-to.md:221]: Added export commands.

**Pull request merged**